test-reflection-cpp.cpp

```diff
@@ -68,6 +68,61 @@ TEST_CASE("nested", "[reflection]")
     CHECK(result == R"(a=1 b=2 c=3 d="hello" e={name="John Doe" email="john@doe.com" age=42})");
 }
 
+TEST_CASE("EnumerateMembers.ind_and_value", "[reflection]")
```

Comment:
```suggestion
TEST_CASE("EnumerateMembers.index_and_value", "[reflection]")
```
Can we somhow agree on ideally fully writing names out (unless very well known abbreviations? `s/ind/index/g` :)

Same for the test case below :)